helpers/dbClient.js

```diff
@@ -0,0 +1,19 @@
+const mongoConnection = "mongodb://127.0.0.1:27017/profile";
+
```

Comment:
any code that deals with MongoClient should basically live here - if you see yourself requiring MongoClient somewhere else, then that's a "code smell", you're breaking the Single Responsibility Principle and you have a module knowing about implementation details (mongo) when it shouldn't.